modules/crypto/src/main/resources/decryption-keys.properties

```diff
@@ -0,0 +1 @@
+V001=<insert decryption key here during deployment>
```

Comment:
Actually, the key name can be anything. To avoid confusion I will change 'V001' to <insert version number here...". Thanks.